config/secrets.yml

```diff
@@ -209,6 +209,11 @@ test:
   sendgrid_marketing_list_id: 10300648
   sendgrid_publishers_list_id: 3986776
   slack_webhook_url: false
+  # Bitflyer env variables
+  bitflyer_host: "https:://localhost/bitflyertest"
```

Comment:
Duplicate secrets and keys, it should read from the environment

Reply:
Not in the test environment, I don't to take anything from the environment here. In fact, now, that you mention it, we should remove all ENV settings from this test section so I'll make a ticket.

Reply:
https://github.com/brave-intl/publishers/issues/3189